### DISCUSSION TITLE
Welcome to maftools Discussions!

### DISCUSSION THREAD

**Discussion by PoisonAlien** [PoisonAlien/maftools#639] [General]
Hello, `maftools` users! This new `GitHub` discussion feature seems pretty useful. 
Use this space to ask questions, general queries, or even feature requests. 
Happy analysis :)

**Comment by ShixiangWang** [comment]
lol ：）

**Comment by esraajamal** [comment]
Hello, 
Thank you very much for this tool, I have a question please, is it possible to plot 2 different maf files in the tcgaCompare ??
